### PR TITLE
[NCL-8340] Update Node to v20.10.0 and NPM to v10.2.5

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,13 @@
...
     "graphology-layout-forceatlas2": "^0.10.1",
     "lodash": "^4.17.21",
-    "pnc-api-types-ts": "project-ncl/pnc-api-types-ts#v2.6.5-pnc-2.6.5",
+    "pnc-api-types-ts": "project-ncl/pnc-api-types-ts#v2.7.0-SNAPSHOT-pnc-8ade3a9",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.12.1",
     "react-scripts": "5.0.1",
     "react-test-renderer": "^17.0.2",
+    "resize-observer-polyfill": "^1.5.1",
```
Comment (matedo1) [inline package.json:30]: Hi @patrikk0123 , why this change is required?

Comment (patrikk0123) [reply package.json:30]: tests were failing

Comment (matedo1) [reply package.json:30]: I am providing error details here:

image

The only one `resize-observer-polyfill` occurence was removed, see `package-lock.json`:

image

**Review by matedo1 — APPROVED** [review]
